### ISSUE TITLE
Option to create new tags is missing (Contao 5.3.9)

### ISSUE THREAD

**Issue by con2net** [numero2/contao-tags#10]
I do not have the option to add new tags via System > Tags. If I enter tags directly via the DB table, they appear here and can also be selected in a news item. 
Have I perhaps just not found the correct way to create them yet or is this a bug?

**Comment by bennyborn** [comment]
This is intentional. Tags are created when they are needed, e.g. when creating a News.
The mask in System > Tags is just there for an overview and for a possible translation 😉

**Comment by con2net** [comment]
Yes, of course that's what I thought at my first try 😉, but entering (new) tags directly in the input line in a news item doesn't work for me. It is not “taken over”. I can only select existing ones.

**Comment by bennyborn** [comment]
It should work, just press enter after adding a new tag 😊

**Comment by con2net** [comment]
Hmm, maybe I have to search for an alternative enter key on my keyboard? :-D

**Comment by bennyborn** [comment]
Enter, Comma and Tab should work.

**Comment by con2net** [comment]
That's what I thought. We previously had the tags extension from Helmut Schottmüller and the input is the same there.
Unfortunately, nothing happens here ...
Tried all the buttons and mouse actions I know ;-)

![numero_tags_1](https://github.com/numero2/contao-tags/assets/13145836/bef3564d-00cd-4528-bfb2-2dd3afe72e66)

**Comment by bennyborn** [comment]
Can you confirm, using the browsers Devtools, that `/bundles/tags/js/backend.js` is successfully loaded?

**Comment by con2net** [comment]
Yes, will be loaded. But ... HTML status code 303

There is a redirection to the 404 page ...?

It may be important to know that we have not set a URL suffix for the pages.

![numero_tags_2](https://github.com/numero2/contao-tags/assets/13145836/e969e2ce-44fd-4870-b510-cfe5d6a3dd40)

**Comment by bennyborn** [comment]
That's the problem right there 😀 
`System > Maintenance > Recreate the symlinks` should fix it.

**Comment by con2net** [comment]
Unfortunately, that doesn't help ... Same problem ...
I have also emptied all other caches.

**Comment by bennyborn** [comment]
So I guess the Symlink in `/public/bundles` is still missing? If so check the permissions and try `composer install` once more.

**Comment by con2net** [comment]
OK, problem solved. The symlink was not there at first. But I saw that - for whatever reason - I had a /web directory and the symlink was set to tags underneath. But my webroot was/is actually /public ... 
I don't know where the /web came from again, but I changed it to public in composer.json and recreated the symlinks. Now there is also a symlink to tags below public and everything works as expected. 
Thanks a lot and sorry that I posted it here and it's not a bug after all.
